Packs/CommonPlaybooks/ReleaseNotes/2_4_7.md

```diff
@@ -0,0 +1,11 @@
+
+#### Playbooks
+
+##### CVE Enrichment - Generic v2
+
+- Updated the playbook output to include several missing command outputs for **VulnDB**, **CIRCL CVE Search**, and **IBM X-Force Exchange** integrations.
+- Replaced the deprecated **'CVE Search'** integration with the **'CIRCL CVE Search'** integration.
```

Comment:
you can remove this as the generic command will be applied

Reply:
since the !CVE Search command is not generic and requires CIRCL integration this is still relevant. also the command used within task number 6 changed to 'cve (CIRCL CVE Search).